src/metabase/models/field_values.clj

```diff
@@ -356,7 +356,9 @@
 
       (and (= (:values field-values) values)
            (= (:has_more_values field-values) has_more_values))
-      (log/debug (trs "FieldValues for Field {0} remain unchanged. Skipping..." field-name))
+      (do
+        (log/debug (trs "FieldValues for Field {0} remain unchanged. Skipping..." field-name))
+        ::fv-skipped)
```

Comment:
This was added for clarity as returning nil was slightly odd, in the future it could be tracked in `update-field-value-stats-count`.